pyproject.toml

```
@@ -16,7 +16,7 @@ snakemake-interface-common = "^1.14.0"
 snakemake-interface-storage-plugins = {git="https://github.com/snakemake/snakemake-interface-storage-plugins.git", branch="dbg/test-storage"}
 boto3 = "^1.33"
 botocore = "^1.33"
-urllib3 = ">=2.0,<2.2"  # https://github.com/boto/botocore/issues/3111#issuecomment-1944524714
+urllib3 = ">=2.0,!=2.2.0"  # https://github.com/boto/botocore/issues/3111#issuecomment-1944524714
```

```suggestion
urllib3 = ">=2.0,!=2.2.0,<2.3"  # https://github.com/boto/botocore/issues/3111#issuecomment-1944524714
```

---

Mhm, actually, since the true dependency is in botocore, we could just mask urllib3 2.2.0 instead of having a min and max constraint here. What do you think?

---

I would prefer no constraints on version that are too conservative. So if an alternative is
```suggestion
urllib3 = "!=2.2.0"  # https://github.com/boto/botocore/issues/3111#issuecomment-1944524714
```
I'd pick it